Tasks/Bulavskaya/Task4/task4.py

```diff
@@ -0,0 +1,54 @@
+while True:
+    length = input('Enter the length (must be >35): ')
+    if length.isdigit() and int(length) > 35:
+        length = int(length)
+        break
+    else:
+        print("Incorerect lenght!")
+
+with open ("text.txt", "r") as f:
+    x = 0
+    res = ""
+    while True:
+        f.seek(x)
+        s = f.read(length)        
+        end1 = s.find("\n")
+
+        if not len(s) == length:
+            string = s[0:]
```

Comment:
btw **s[0:]** is the same as **s[:]** and it means "s from the start till the end", so this line is the same as **string = s**